README.md

```diff
@@ -133,6 +133,12 @@ ManagedClusterAddOn resource. This will enable changes to the ManifestWork on th
 persist, but direct changes to resources on a managed cluster will still be reverted to match the
 ManifestWork.
 
+### Image Override Troubleshooting
+If there is trouble overriding an image or other configurations in ACM, the `klusterletaddonconfig-pause` annotation is required on the `KlusterletAddonConfig` to add the `addon.open-cluster-management.io/values` annotation overrides in the `ManagedClusterAddon`.
```

Comment:
Could you also add newlines before and after the paragraph and add some line wrapping? It matches formatting elsewhere here. (And I think VSCode might also do this for you if you use the Prettier extension.)
```suggestion
### Image Override Troubleshooting

If there is trouble overriding an image or other configurations in ACM, the `klusterletaddonconfig-pause` annotation is required on the `KlusterletAddonConfig` to add the `addon.open-cluster-management.io/values` annotation overrides in the `ManagedClusterAddon`.

```

Reply:
Prettier for readme?

Reply:
Yes--Prettier for Markdown, specifically.